docs/modules/ROOT/pages/admin.adoc

```diff
@@ -185,6 +185,34 @@ From then on, your community can use any Governor compatible voting DApp (such a
 
 image::defender-admin-governor-track.png[Defender tracks the state of your proposal by querying the Governor]
 
+[[Fireblocks]]
+== Fireblocks
+
+You can also submit transactions to Fireblocks from Defender directly. https://www.fireblocks.com/[Fireblocks] is an asset management solution that leverages multi-party computation for securing all treasury operations.
```

Comment:
Lets add target blank for all links so user are not redirected fro. our page.
https://www.fireblocks.com/[Fireblocks, window=_blank]